app/src/test/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepositorySessionsTest.kt

```diff
@@ -47,67 +47,89 @@ class AppRepositorySessionsTest {
-            highlight = false
-            changedIsCanceled = false
-        }
-
-        private val SESSION_2002 = createSession("2002").apply {
-            highlight = true
-            changedIsCanceled = false
-        }
-
-        private val SESSION_2003 = createSession("2003").apply {
-            highlight = true
-            changedIsCanceled = true
-        }
-
-        private val SESSION_2004 = createSession("2004").apply {
-            highlight = false
-            changedIsCanceled = true
-        }
-
-        private val SESSION_3001 = createSession("3001").apply {
-            changedIsCanceled = false
-        }
-
-        private val SESSION_3002 = createSession("3002").apply {
-            changedIsCanceled = true
-        }
-
-        private fun createSession(sessionId: String) = Session(sessionId).apply {
-            url = "" // only initialized for toSessionsDatabaseModel()
-        }
+        private val SESSION_1001 = createSession(
+            sessionId = "1001",
+            changedIsCanceled = false,
+            changedTitle = false,
+            changedIsNew = false,
+        )
+
+        private val SESSION_1002 = createSession(
+            sessionId = "1002",
+            changedIsCanceled = true,
+            changedTitle = false,
+            changedIsNew = false,
+        )
+
+        private val SESSION_1003 = createSession(
+            sessionId = "1003",
+            changedIsCanceled = false,
+            changedTitle = true,
+            changedIsNew = false,
+        )
+
+        private val SESSION_1004 = createSession(
+            sessionId = "1004",
+            changedIsCanceled = false,
+            changedTitle = false,
+            changedIsNew = true,
+        )
+
+        private val SESSION_1005 = createSession(
+            sessionId = "1005",
+            changedIsCanceled = true,
+            changedTitle = true,
+            changedIsNew = true,
+        )
+
+        private val SESSION_2001 = createSession(
+            sessionId = "2001",
+            highlight = false,
+            changedIsCanceled = false,
+        )
+
+        private val SESSION_2002 = createSession(
+            sessionId = "2002",
+            highlight = true,
+            changedIsCanceled = false,
+        )
+
+        private val SESSION_2003 = createSession(
+            sessionId = "2003",
+            highlight = true,
+            changedIsCanceled = true,
+        )
+
+        private val SESSION_2004 = createSession(
+            sessionId = "2004",
+            highlight = false,
+            changedIsCanceled = true,
+        )
+
+        private val SESSION_3001 = createSession(
+            sessionId = "3001",
+            changedIsCanceled = false,
+        )
+
+        private val SESSION_3002 = createSession(
+            sessionId = "3002",
+            changedIsCanceled = true,
+        )
+
+        private fun createSession(
```

Comment:
This feels like a needless indirection now. The `Session` constructor could be used directly.

Reply:
:x: I like to keep the helper function to at least not repeat the `url` statement over and over.

Reply:
The value for the `url` parameter is the same as the default value. So it's not really doing anything.

Reply:
Interesting. Simplified now. :heavy_check_mark: